roles/kubernetes/preinstall/tasks/0040-set_facts.yml

```diff
@@ -131,16 +131,18 @@
         {{ [ skydns_server ] + [ skydns_server_secondary ] }}
       {%- elif dns_mode == 'manual' and not dns_early|bool -%}
         {{ ( manual_dns_server.split(',') | list) }}
+      {%- elif dns_mode == 'none' and not dns_early|bool -%}
```

Comment:
Do you really need this line if you already add a default value to [] under line 143/145

Reply:
It seems that intention for this block was to cover possibilities for dns_mode, with mentioning it here I think it is more readable. These conditions may not be exhausting list of possibilities, there is no 'else' leg, so there is additional default filter in place where we use coredns_server. Later all the addends has this filter applied, I followed this convention, which made 'else' redundant here and I did not defined it. It similar to 'cloud_resolver' fact, but there is 'else' which I think make default filter unnecessary. For me most readable, having jinja templates already in use, would be put exhausting list of possibilities in place where we define fact, and not use default filter later.